examples/image_retraining/retrain.py

```diff
@@ -964,10 +964,32 @@ def export_model(module_spec, class_count, saved_model_dir):
     )
 
 
+def logging_level_verbosity(logging_verbosity):
+    """Converts logging_level into TensorFlow logging verbosity value
```

Comment:
https://www.tensorflow.org/community/contribute/code_style asks to indent by 2 spaces per level.

Reply:
fixed